src/oop_ext/foundation/callback/_callbacks.py

```diff
@@ -1,6 +1,6 @@
 from typing import Callable, Any, List, Tuple
```

Comment:
This module doesn't have type hints in almost all input parameters =/

Reply:
Added `mypy: disallow-untyped-defs` to the top of the module and fixed the few places where they were missing... what do you mean by "This module doesn't have type hints in almost all input parameters =/"? All of them do now, actually.

Reply:
On this file `_callbacks.py` the method `Before`, Regiter` and `__exit__` doesn't have types.

Reply:
Ahh my bad, I confused `_callback.py` and `_callbacks.py` (very unfortunate that they are named so closely).

Thanks, you pointing this out actually brought out that there was a serious bug in the signature itself, see 022cce0.